packages/@aws-cdk/core/README.md

```diff
@@ -35,6 +35,30 @@ Duration.days(7)        // 7 days
 Duration.parse('PT5M')  // 5 minutes
 ```
 
+## Storage Quantities
+
+To make specification of digital storage quantities unambiguous, a class called
+`Storage` is available.
+
+An instance of `Storage` is initialized through one of its static factory methods:
+
+```ts
+Storage.kibibytes(200) // 200 KiB
```

Comment:
Cool!

I wonder about the name, wondering if `Size.kibibytes()` wouldn't be better. 

But yeah!

Reply:
I like it. Size definitely sounds better.